StandardSocketsHttpHandler/StandardSocketsHttpHandler.csproj

```diff
@@ -7,6 +7,9 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <Version>2.2.0.5</Version>
     <Authors>Tal Aloni</Authors>
+    <Version>2.2.0.8-debug-2</Version>
+    <Authors>Tal Aloni, Aristarkh Zagorodnikov</Authors>

```

Comment:
One PR and suddenly you're a co-author?

Reply:
The change to this file is accidental, I failed to correctly isolate changes from our [fork](https://github.com/drivenet/StandardSocketsHttpHandler), where I'm indeed a co-author.